### ISSUE TITLE
Hope this tool into pypi

### ISSUE THREAD

**Issue by leafonsword** [mahmoudadel2/pysed#2]
pysed is an awesome tool, hope this tool could into pypi, so user could install using pip;
but pypi has already had an same name tool, so I think your tool could be renamed to other name like: sed-python、sedy.......

**Comment by Jambon1510** [comment]
Agree, I installed pysed but it was referring to https://pypi.org/project/pysed/ and this is not usable within python code
